src/lib/styles/app.css

```
@@ -1,8 +1,21 @@
-@tailwind utilities;
+@import 'tailwindcss' source('../../../src');
+@plugin '@tailwindcss/typography';
+@plugin 'daisyui' {
+    prefix: 'dy-';
+    themes: light --default;
+}
 
-@layer base {
+@source inline("grid-cols-{{1..12..1},none,subgrid}");
+
+/*@config "../../../tailwind.config.cjs";*/
```

Stylelint (17.13.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)